.github/FUNDING.yml

```diff
@@ -1 +1 @@
-custom: ["paypal.me/yaichenbaum"]
+github: [yaichenbaum]
```

Comment:
Was this change intentional?

Reply:
@winston-de I merged the from latest main and then did a squash commit. This is not part of my changes, must be from the last commit to main.